packages/http/lib/decorators.tsp

```diff
@@ -80,7 +107,7 @@ extern dec query(target: ModelProperty, queryNameOrOptions?: string | QueryOptio
  * op read(@path explicit: string, implicit: string): void;
  * ```
  */
-extern dec path(target: ModelProperty, paramName?: valueof string);
+extern dec path(target: ModelProperty, paramNameOrOptions?: valueof string | PathOptions);
```

Comment:
Need to add `explode: true` for query parmaeters and deprecate teh format: option